File: 1550-three-consecutive-odds/1550-three-consecutive-odds.cpp
class Solution {
public:
    bool threeConsecutiveOdds(vector<int>& arr) {
        int count = 0, n = arr.size();
        for(int i = 0; i < n - 2; i++){
            if(arr[i] % 2 == 1 && arr[i + 1] % 2 == 1 && arr[i + 2] % 2 == 1){
                return true;
            }
        }
        return false;
    }
};

File: 1550-three-consecutive-odds/1550-three-consecutive-odds.py
class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        for i in range(0, len(arr)):
            if(arr[i] % 2 == 1 and arr[i + 1] % 2 == 1 and arr[i + 2] % 2 == 1):
                return True
            